docs/core/extensions/globalization-icu.md

```diff
@@ -50,7 +50,16 @@ The following table shows which versions of .NET are capable of loading the ICU
 
 ### Behavioral differences
 
-If you upgrade your app to target .NET 5 or later, you might see changes in your app even if you don't realize you're using globalization facilities. This section lists one of the behavioral changes you might see, but there are others too.
+If you upgrade your app to target .NET 5 or later, you might see changes in your app even if you don't realize you're using globalization facilities. This section lists several behavioral changes you might see, but there are others too.
+
+#### String Sorting and System.Globalization.CompareOptions
+
+`CompareOptions` is the options enumeration which can be passed to `String.Compare` to influence how two strings are compared.
+
+Comparing strings for equality and determining their sort order differs between NLS and ICU. In particular:
+
+- The default string sort order is different, so this will be apparent even if you do not use `CompareOptions` directly. When using ICU, the `None` default option performs the same as `StringSort`. `StringSort` sorts non-alphanumeric characters before alphanumeric ones (so "bill's" sorts before "bills", for example). To restore the previous `None` functionality, you must use the NLS-based implementation.
```

Comment:
```suggestion
- The default string sort order is different, so this will be apparent even if you don't use `CompareOptions` directly. When using ICU, the `None` default option performs the same as `StringSort`. `StringSort` sorts non-alphanumeric characters before alphanumeric ones (so "bill's" sorts before "bills", for example). To restore the previous `None` functionality, you must use the NLS-based implementation.
```